.circleci/config.yml

```diff
@@ -10,7 +10,9 @@ workflows:
       - build:
           matrix:
             parameters:
-              ruby-version: ["2.6.8", "2.7.4", "3.0.2"]
+              ruby-version: ["3.0.2", "3.1.2"]
```

Comment:
All our services have already migrated to Ruby 3, right?

Reply:
And maybe add `spec.required_ruby_version = '>= 3.0.0'` to gemspec?

Reply:
:+1: